yarn.lock

```diff
@@ -1004,7 +1004,7 @@
   dependencies:
```

Comment:
We already have a `package-lock.json` file

Reply:
removed it